e2e/features/start-appeal-out-of-country-no-sponsor.feature

```diff
@@ -0,0 +1,94 @@
+Feature: Out of country appeal with no sponsor
+
+  Background:
+    Given I am signed in as a `Legal Rep`
+    And I create a new case
+    And I complete the `Tell us about your client out of country` page
+    And I select `No` for the `Is your client currently living in the United Kingdom?` field
+    And I click the `Continue` button
+
+  @start-appeal-out-of-country-no-sponsor-no-address @RIA-2110
+  Scenario: Select No address
+
+    And I complete the `Decision type` page
+    And I complete the `Entry clearance decision details` page
+    And I complete the `Upload the Notice of Decision` page
+    And I complete the `Basic details` page
+    And I complete the `Tell us about your client's nationality` page
+
+    Given I am on the `Your client's address` page
+    When I select `No` for the `Does your client have a correspondence address outside the UK?` field
+    And I click the `Continue` button
+    And I complete the `The appellant's contact preference` page
+    And I complete the `Sponsor` page
+    And I complete the `Type of appeal` page
+    And I complete the `The grounds of your appeal` page
+    And I complete the `New matters` page
+    And I complete the `Has your client appealed against any other UK immigration decisions?` page
+    And I complete the `Legal representative details` page
+    And I complete the `Start appeal check your answers` page
+
+    Then I should see the text `Your appeal details have been saved`
+    And I should see the text `You still need to submit it`
+    And I should see the text `If you're ready to proceed submit your appeal.`
+    And I should see the text `Not ready to submit yet?`
+    And I should see the text `You can return to the case details to make changes.`
+    And I click the `Close and Return to case details` button
+
+    When I click the `Overview` tab
+    Then I should see `DRAFT` for the `Appeal reference` field
+    And I should see `Yes` for the `Out of country` field
+    And I should see `GWF1234567` for the `Global Web form` field
+    And I should see `{$TODAY|D MMM YYYY}` for the `Date of entry clearance decision` field
+
+    When I click the `Appeal` tab
+    Then I should see `DRAFT` for the `Appeal reference` field
+    And I should see `Yes` for the `Out of country` field
+    And I should see `A decision to refuse a human rights claim for entry clearance` for the `Out of country decision` field
+
```

Comment:
@mkayad This Test won't work with when we switch ON F&P Flag, AC is covering F&P scenarios as well and Not 
sure why we are not submitting appeal as part of this Tests.